magicctapipe/scripts/lst1_magic/semi_automatic_scripts/clusters.py

```diff
@@ -0,0 +1,41 @@
+"""
+Module for generating bash script lines for running analysis in different clusters
+"""
+
+
+def slurm_lines(p, J, array=None, mem=None, out_err=None):
```

Comment:
variable names here could be a bit different and more descriptive:

- p --> queue
- J --> job_name
- out_err --> out_name (since it is used for both out and err, just with different extension)

Reply:
I thought first to use such descriptive names, but since either way we describe them in the doc string of the function I used directly the names that are used in slurm. But if you prefer I can change them to queue, job_name and out_name

Reply:
I would prefer if they are changed :) even if they are function parameters, it is still better to use descriptive names

Reply:
ok, done